### PR TITLE
#RI-3478 - automatic update dependencies versions

### PR DESCRIPTION
**Pull request by zalenskiSofteq**
#RI-3478 - automatic update dependencies versions

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1044,6 +1075,8 @@ workflows:
   # Workflow for feature, bugfix, main branches
   feature-main-branch:
     jobs:
+      - licenses-check:
```
Comment (arthosofteq) [inline .circleci/config.yml:1078]: I think we must move it after approve

File: .circleci/config.yml [hunk 2]
```diff
@@ -1414,3 +1449,16 @@ workflows:
...
+  triggers:
+    - schedule:
+        cron: '0 0 * * 1'
+        filters:
+          branches:
+            only:
+              - main
+  jobs:
+    # Process all licenses
+    - licenses-check:
```
Comment (arthosofteq) [inline .circleci/config.yml:1463]: also if we have such checks on weekly basis, why do we need it for feature/main branches?

**Review by arthosofteq — DISMISSED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (zalenskiSofteq) [reply .circleci/config.yml]: It's run weekly very rare, let it be automatic to don't forget about this

File: .circleci/config.yml (hunk 2 above)
Comment (zalenskiSofteq) [reply .circleci/config.yml:1461]: It was for test, removed

**Review by vlad-dargel — APPROVED** [review]

**Review by vlad-dargel — APPROVED** [review]

**Review by arthosofteq — APPROVED** [review]

**Pull request merged**